src/Components/CardsContainer.js

```diff
@@ -0,0 +1,29 @@
+import SingleCard from "./SingleCard";
+import CardGroup from "react-bootstrap/CardGroup";
+import Container from "react-bootstrap/Container";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
```

Comment:
This is cleaner, I checked and it works.
```suggestion
import {CardGroup, Container, Row, Col} from "react-bootstrap";
```